fs/fsshrink.py.data/test-shrink.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#author : Miklos Szeredi <mszeredi@redhat.com>
```

Comment:
Do we have author's approval of hosting it here ?

Reply:
i think should be fine as it in lkml.org and open source

Reply:
@PraveenPenguin Can this be pushed to LTP instead ?

Reply:
@narasimhan-v i try to  but not sure about acceptance , will discuss more offline with you